classes/Validator.js

```diff
@@ -1,6 +1,10 @@
+import Ajv from './Ajv.js';
+
 class Validator {
     constructor(...args) {
-        let Ajv;
+        if (!Ajv) {
+            throw new Error('Ajv class is required.');
```

Comment:
That's not necessary as the import will fail before hitting this code if it's not defined

Reply:
This will give a better error log if `Ajv` exports an empty value. For example if the `Ajv` source is not included or override. Perhaps I should check if it's a constructor instead but this check is indeed important for clarity and debugging ;)